### PR TITLE
Fix translations for sem.asyl

### PR DESCRIPTION
**Pull request by Sgachet**
Small correction about the translation of the title

### CODE REVIEW

File: chsdi/locale/en/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -4940,6 +4940,9 @@ msgstr "Nuclear Power Plants"
 msgid "ch.bfe.kleinwasserkraftpotentiale"
 msgstr "Water power potential"
 
+msgid "ch.bfe.ladestellen-elektromobilitaet"
+msgstr "Ladeinfrastruktur Elektromobilität"
```
Comment (oterral) [inline chsdi/locale/en/LC_MESSAGES/chsdi.po]: No translation ?

File: chsdi/locale/fr/LC_MESSAGES/chsdi.po [hunk 2]
```diff
@@ -4940,6 +4940,9 @@ msgstr "Centrales nucléaires"
 msgid "ch.bfe.kleinwasserkraftpotentiale"
 msgstr "Potentiel petite hydraulique"
 
+msgid "ch.bfe.ladestellen-elektromobilitaet"
+msgstr "Ladeinfrastruktur Elektromobilität"
```
Comment (oterral) [inline chsdi/locale/fr/LC_MESSAGES/chsdi.po]: No translation?

File: chsdi/locale/it/LC_MESSAGES/chsdi.po [hunk 3]
```diff
@@ -4940,6 +4940,9 @@ msgstr "Centrali nucleari"
 msgid "ch.bfe.kleinwasserkraftpotentiale"
 msgstr "Potenz. piccole centr. idraul."
 
+msgid "ch.bfe.ladestellen-elektromobilitaet"
+msgstr "Ladeinfrastruktur Elektromobilität"
```
Comment (oterral) [inline chsdi/locale/it/LC_MESSAGES/chsdi.po]: No translation?

File: chsdi/locale/en/LC_MESSAGES/chsdi.po (hunk 1 above)
Comment (Sgachet) [reply chsdi/locale/en/LC_MESSAGES/chsdi.po]: @AFoletti It's correct ?

File: chsdi/locale/it/LC_MESSAGES/chsdi.po (hunk 3 above)
Comment (AFoletti) [reply chsdi/locale/it/LC_MESSAGES/chsdi.po]: ch.bfe.ladestellen-elektromobilitaet is the DIEMO layer and only on dev. No problem with that

**Pull request merged**